### PR TITLE
Safeguards setting of env var to prevent exception when value is None

### PR DESCRIPTION
**Pull request by benjamin-hg**
@bastianeicher as it can be quite cumbersome to find out which env var is causing the problem, I suggest to add some output and avoid the exception if value is `None`

### CODE REVIEW

**Comment by bastianeicher** [comment]
The error message indicating which env var is causing the problem is great.
However, I think the program should still crash (exception or non-zero exit code). Otherwise the problem might go unnoticed in a lot of CI output.

**Comment by benjamin-hg** [comment]
I thought the actual checks of environment variables should be done by helmfile right afterwards: `requireEnv` would yield an error if the variable is not set. Am I missing something? :thinking:

**Comment by bastianeicher** [comment]
If an environment variable is intentionally unset it should always be either completely absent from the input JSON or set to an empty string (`""`). A `null` value always indicates that something went wrong (e.g., in our case incorrect output generated by a TFS task).

Relying on `requiredEnv` in helmfile to catch this would still let cases slip through were an optional env var (like a password with an unsafe default value) becomes unintentionally unset.

File: src/helmfile.py [hunk 1]
```diff
@@ -22,6 +22,6 @@ def delete(environment, helmfilePath, purge=False):
 def apply_environ(environment):
     for key, value in environment.items():
         if value is None:
-            print('\nUnable to set env var %s -- value is missing' % (key))
+            raise TypeError('\nUnable to set env var %s -- value is None' % (key))
```
Comment (benjamin-hg) [inline src/helmfile.py]: @bastianeicher or better use `ValueError`?

Comment (bastianeicher) [reply src/helmfile.py]: `TypeError` seems good to me.
The leading `\n` in the exception message should probably be removed though.

**Comment by bastianeicher** [comment]
Thanks. :) Merged.

**Pull request merged**